broker/src/main/java/io/zeebe/broker/system/configuration/DataCfg.java

```diff
@@ -45,11 +46,24 @@ public void setDirectories(final List<String> directories) {
     this.directories = directories;
   }
 
+  public Long getLogSegmentSizeInBytes() {
```

Comment:
It's a little strange imo to have a `Long` but I get the idea. Maybe once we migrate the config we can revisit this and ensure we always have some value here.

Reply:
Agreed. The caller assumed that the String might be null, that's why I did it this way.